app/src/main/res/layout/administrator_controls_general_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
```

Comment:
Remove this. This is not getting used.

Reply:
I cannot remove for above mentioned reason